zulip/integrations/bridge_with_irc/irc_mirror_backend.py

```diff
@@ -20,17 +20,19 @@ def __init__(
         server: str,
         nickserv_password: str = "",
         port: int = 6667,
+        sasl_login: str = "",
     ) -> None:
         self.channel = channel  # type: irc.bot.Channel
         self.zulip_client = zulip_client
         self.stream = stream
         self.topic = topic
         self.IRC_DOMAIN = server
         self.nickserv_password = nickserv_password
+        self.sasl_login = sasl_login
         # Make sure the bot is subscribed to the stream
         self.check_subscription_or_die()
         # Initialize IRC bot after proper connection to Zulip server has been confirmed.
-        irc.bot.SingleServerIRCBot.__init__(self, [(server, port)], nickname, nickname)
+        irc.bot.SingleServerIRCBot.__init__(self, [(server, port)], nickname, nickname, sasl_login)
```

Comment:
Can you pass `sasl_login` as a keyword argument? See https://github.com/jaraco/irc/blob/f1bc7baaad0892ddbd1ba38491635f00237a2fd9/irc/bot.py#L158.

Reply:
I tried passing `sasl_login` as keyword argument and have updated the code. 

But the following error occurs:
```
Traceback (most recent call last):
  File "/home/sbansal1999/p/python-zulip-api/zulip/integrations/bridge_with_irc/irc-mirror.py", line 70, in <module>
    bot.start()
  File "/srv/zulip-py3-venv/lib/python3.10/site-packages/irc/bot.py", line 347, in start
    self._connect()
  File "/srv/zulip-py3-venv/lib/python3.10/site-packages/irc/bot.py", line 187, in _connect
    self.connect(
  File "/home/sbansal1999/p/python-zulip-api/zulip/integrations/bridge_with_irc/irc_mirror_backend.py", line 47, in connect
    self.c = self.reactor.loop.run_until_complete(self.connection.connect(*args, **kwargs))
TypeError: AioConnection.connect() got an unexpected keyword argument 'sasl_login'
```

Reply:
That actually doesn't make sense. The `AioConnection` is a subclass of `ServerConnection`. https://github.com/jaraco/irc/pull/199 modifies `ServerConnection`. So the `AioConnection` should accept `sasl_login`.